lib/android/src/main/java/com/wix/reactnativeuilib/wheelpicker/ThemedNumberPicker.java

```diff
@@ -0,0 +1,25 @@
+package com.wix.reactnativeuilib.wheelpicker;
```

Comment:
@bogobogo 
What is this file? What's ThemedNumberPicker for ?

Reply:
NumberPicker, the original native android component, does not expose an api to change the style of the divider, and its text inputs. One way to solve this is to use reflection. The other is to create a "theme" of styles for the app, that "trickles down" (i don't really know how it works under the hood) to the the component it self. Since I don't want to create a theme for the app, i wrapped the component with a theme where I define the styles. These themes are than implemented only on this component